src/modelinstance.hpp

```diff
@@ -218,6 +218,7 @@ class ModelInstance {
          */
     Status loadOVModelUsingCustomLoader();
 
+public:
     template <typename RequestType>
     const Status validate(const RequestType* request);
```

Comment:
Why do we need it to be public now?

Reply:
This is temporary public to add possibility to use it in kfs_grpc_inference_service without ModelInstance::infer implemented. Will add TODO to remove this line